examples/meta/src/evaluation/multiclassaccuracy.sg.in

```diff
@@ -1,9 +1,9 @@
-File ground_truth = csv_file("@SHOGUN_DATA@/label_train_multiclass.dat")
-File predicted = csv_file("@SHOGUN_DATA@/label_train_multiclass.dat")
+File ground_truth = csv_file("@SHOGUN_DATA@/classifier_binary_2d_linear_labels_train.dat")
```

Comment:
nice thanks!
Another thing is the filename of the example. Could you make it "accuracy_multiclass"?

And could you also add a data file? (This can also be used to confirm whether you get a good accuracy with the model here and the example is not nonsense :) )

Reply:
Will do, but will have to upload the file tomorrow since my internet is so bad atm :S

Reply:
there is no rush